ext/mbstring/tests/mb_convert_variables.phpt

```diff
@@ -17,19 +17,19 @@ $sjis = base64_decode('k/qWe4zqg2WDTINYg2eCxYK3gUIwMTIzNIJUglWCVoJXgliBQg==');
 // JIS string (BASE64 encoded)
 $jis = base64_decode('GyRCRnxLXDhsJUYlLSU5JUgkRyQ5ISMbKEIwMTIzNBskQiM1IzYjNyM4IzkhIxsoQg==');
 // EUC-JP string
-$euc_jp = '���ܸ�ƥ����ȤǤ���01234������������';
```

Comment:
Hmm, interesting to see that GitHub is able to recognize this EUC-JP encoded string and display it correctly here...

Reply:
I agree that it is preferable to have *.phpt files encoded as UTF-8 whenever possible.